resources/index.html

```diff
@@ -109,50 +116,82 @@
 		</style>
 	</head>
 
+
 	<body aria-label="">
 		<noscript title="No JavaScript Support">
 			<h1>You need to enable JavaScript to run this app.</h1>
 		</noscript>
-		<div id='load-spinner' aria-label="loading">
+		<!-- <div id='load-spinner' aria-label="loading">
 			<div class="lds-roller"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div></div>
-		</div>
+		</div> -->
```

Comment:
Loader is disabled by this comment.

Reply:
I see, the issue is how we delete the element after the vscode-web loaded in browser.

This patch will work:


```
diff --git a/resources/index.html b/resources/index.html
index 9f05f8b..c014364 100644
--- a/resources/index.html
+++ b/resources/index.html
@@ -121,12 +121,42 @@
 		<noscript title="No JavaScript Support">
 			<h1>You need to enable JavaScript to run this app.</h1>
 		</noscript>
-		<!-- <div id='load-spinner' aria-label="loading">
+		<div id='load-spinner' aria-label="loading">
 			<div class="lds-roller"><div></div><div></div><div></div><div></div><div></div><div></div><div></div><div></div></div>
-		</div> -->
+		</div>
 	</body>
 
 	<script>
+		function waitForElement(selector) {
+			return new Promise(function(resolve, reject) {
+				const element = document.querySelector(selector);
+
+				if(element) {
+				resolve(element);
+				return;
+				}
+
+				const observer = new MutationObserver(function(mutations) {
+				mutations.forEach(function(mutation) {
+					const nodes = Array.from(mutation.addedNodes);
+					for(const node of nodes) {
+					if(node.matches && node.matches(selector)) {
+						observer.disconnect();
+						resolve(node);
+						return;
+					}
+					};
+				});
+				});
+
+				observer.observe(document.documentElement, { childList: true, subtree: true });
+			});
+		}
+		waitForElement('div[role="application"]').then(function() {
+			document.querySelector('#load-spinner').remove();
+		});
+
+
 
 parseGitHubUrl = (url) => {
 			const urlObj = new window.URL(url);
```